soccer/gameplay/plays/restarts/placement.py

```diff
@@ -59,8 +59,8 @@ def on_enter_placing(self):
                                  required=True,
                                  priority=90)
 
-        #line_up = tactics.line_up.LineUp(self.create_lineup())
-        #self.add_subbehavior(line_up, 'line_up', required=True, priority=80)
+        line_up = tactics.line_up.LineUp(self.create_lineup())
+        self.add_subbehavior(line_up, 'line_up', required=True, priority=80)
```

Comment:
The manual says

>When the ball goes out of play, robots should remain 500mm from the ball as the ball is placed until the restart signal is given by the referee.

I would assume the 'restart' signal would be the signal that is given during normal restarts (ie: when the ref places the ball themselves). That means we would have a little bit of time since the opposing team needs to get set up for a kick, but we won't have more than 5s or so, tops. 

This is a very new rule, so I would hope the refs would be lenient and give teams time to adjust.

I would reccomend placing the robots in a general defensive line (in relation to the target of the move command), perhaps, so they are close to their intended target places when resuming.